app/controllers/groups_controller.rb

```diff
@@ -76,6 +76,16 @@ def reject_pending
     handle_action(path: draw_group_path(@draw, @group), **result)
   end
 
+  def leave
+    membership = current_user.membership
+    result = if membership
```

Comment:
Hmm, that's a good question. I guess we don't - if the current user is in `group.members` it _must_ have an accepted membership. I'll update this, thanks!